src/record.h

```diff
@@ -16,7 +16,9 @@ class Record
 public:
   enum EFlag {
     FLAG_MASTER = 0x00000001,
+    FLAG_LIGHT_ALTERNATE = 0x00000100,
     FLAG_LIGHT = 0x00000200,
+    FLAG_OVERRIDE = 0x00000200,

```

Comment:
Are these two mean to be the same?

Reply:
Yes.

In SSE and FO4 the x200 flag means it's a light plugin. But in SF, x200 is a new 'override' plugin flag type and x100 is a light plugin.

Reply:
Ah I see. Might be worth to add this information as comments